File: Java/KasirMini.java
import java.util.Scanner;

public class KasirMini {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        float harga = input.nextFloat();

        System.out.print("Masukkan jumlah beli: ");
        int jumlah = input.nextInt();

        float total = harga * jumlah;
        float diskon;

        if (total > 10900) {
            diskon = total * 0.9f;   // sesuai contoh
        } else {
            diskon = 0;
        }

        float total_bayar = total - diskon;

        System.out.println("\n===== Struk Pembayaran =====");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Satuan  : " + harga);
        System.out.println("Jumlah Beli   : " + jumlah);
        System.out.println("Total Harga   : " + total);
        System.out.println("Diskon        : " + diskon);
        System.out.println("Total Bayar   : " + total_bayar);
        System.out.println("============================");
    }
}

File: PHP/Kasir_mini.php
<?php

$nama = readline("Masukkan nama barang: ");
$harga = (float)readline("Masukkan harga barang: ");
$jumlah = (int)readline("Masukkan jumlah beli: ");

$total = $harga * $jumlah;

if ($total > 10900) {
    $diskon = $total * 0.09;   // sesuai contoh
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

echo "\n===== Struk Pembelian =====\n";
echo "Nama Barang   : $nama\n";
echo "Harga Satuan  : $harga\n";
echo "Jumlah Beli   : $jumlah\n";
echo "Total Harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total Bayar   : $total_bayar\n";
echo "============================\n";
?>